### PR TITLE
#523 Terms of use

### PR DESCRIPTION
**Pull request by bttendulkara**
please review issue #1.

### CODE REVIEW

File: src/main/asciidoc/en/Ch01_TermsOfUse.adoc [hunk 1]
```diff
@@ -19,7 +19,35 @@ In order to use this document, you are required to agree to abide by the followi
...
+
+In order to use this document, you are required to agree to abide by the following terms. If you do not agree with the terms, you must immediately delete or destroy this document and all its duplicate copies.
+
+. Copyrights and all other rights of this document shall belong to Nippon Telegraph and Telephone Corporation (hereinafter referred to as "NTT")  or third party possessing such rights.
+
+.  This document may be reproduced, translated or adapted, in whole or in part for personal use. However, deletion of the terms given on this page and copyright notice of NTT is prohibited.
+
+. This document may be changed, in whole or in part for personal use. Creation of secondary work using this document is allowed. However, “Reference document: Macchinetta Batch Framework Development Guideline” or equivalent documents may be mentioned in created document and its duplicate copies.
+
+. Document and its duplicate copies created according to Clause 2 may be provided to third party only if these are free of cost.
```
Comment (btundes) [inline src/main/asciidoc/en/Ch01_TermsOfUse.adoc]: @bttendulkara L-42
Wrong usage--> free of cost.
Correct usage--> open source.

**Comment by btundes** [comment]
@bttendulkara please provide the reference of english issue in above PR Comment.
I have also added a comment to the translated text.
Request you to update it accordingly.

**Comment by bttendulkara** [comment]
@btundes Review comment incorporated

**Comment by btundes** [comment]
@bttendulkara please provide the reference of english issue in above PR Comment.

**Comment by btundes** [comment]
@bttendulkara I have confirmed the changes and does not have any further comment.

**Pull request merged**